ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.16.yaml

```diff
@@ -65,8 +65,7 @@ tests:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      ENABLE_DES_DEFAULT_MACHINE: "true"
-    workflow: cucushift-installer-rehearse-azure-ipi-disk-encryption-set-minimal-permission
+    workflow: cucushift-installer-rehearse-azure-ipi-minimal-permission
```

Comment:
it's configured as ipi-minimal-permission workflow here

Reply:
yes, I want to verify ipi-minimal-permission also works well after the change in step `azure-provision-service-principal-minimal-permission`.

But [job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/49747/rehearse-49747-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.16-installer-rehearse-azure/1767516821574914048) failed as API is unavailable after bootstrap is removed, similar as bug OCPBUGS-29429.
```
W0312 12:35:39.943973     210 reflector.go:539] k8s.io/client-go/tools/watch/informerwatcher.go:146: failed to list *v1.ClusterVersion: Get "https://api.ci-op-9dv0mywm-df3f3.qe.azure.devcluster.openshift.com:6443/apis/config.openshift.io/v1/clusterversions?fieldSelector=metadata.name%3Dversion&limit=500&resourceVersion=0": dial tcp 52.162.255.183:6443: i/o timeout
```